assets/js/ubi.js

```diff
@@ -0,0 +1,302 @@
+
+
+
+
+function guiid() {
```

Comment:
This method always struck me as an odd name.   It's generading a GUID right?    `guid`?